pkg/controller/perconaservermongodb/version.go

```diff
@@ -80,14 +88,24 @@ func (r *ReconcilePerconaServerMongoDB) sheduleEnsureVersion(cr *api.PerconaServ
 		return err
 	}
 
-	r.crons.jobs[jobName] = Shedule{
+	log.Info(fmt.Sprintf("add new job (%s): %s", jobName(cr), cr.Spec.UpgradeOptions.Schedule))
```

Comment:
why you are using sprintf?
log.Info already contains keys and values stuff
Like `log.Info("add new job","job name",jobName(cr),"schedule",cr.Spec.UpgradeOptions.Schedule)`

Reply:
Fixed as a part of https://github.com/percona/percona-server-mongodb-operator/pull/472/commits/9a8a46cec1d2a2523f4f9cf9fe19fb31147d5ca4